lib/ast.js

```diff
@@ -335,6 +335,26 @@ var AST_Toplevel = DEFNODE("Toplevel", "globals", {
             }
         }));
         return wrapped_tl;
+    },
+    wrap_enclose: function(arg_parameter_pairs) {
+        var self = this;
+        var args = [];
+        var parameters = [];
+
+        arg_parameter_pairs.forEach(function(pair) {
+            var splitAt = pair.lastIndexOf(":");
+
```

Comment:
Remove blank line.

Reply:
uh, I just copy this snippet from v2.x branch, but remove some blank line it's not a bad idea : )